kubernetes.go

```diff
@@ -549,6 +562,41 @@ func (svc *KubernetesServiceOp) DeleteNodePool(ctx context.Context, clusterID, p
 	return resp, nil
 }
 
+func boolToQuery(b bool) string {
+	if b {
+		return "1"
+	}
+
+	return "0"
+}
+
+// DeleteNode deletes a specific node in a node pool.
+func (svc *KubernetesServiceOp) DeleteNode(ctx context.Context, clusterID, poolID, nodeID string, deleteReq *KubernetesNodeDeleteRequest) (*Response, error) {
```

Comment:
I think this new name is going to confuse people.  We refer to it as recycle in the control plan and the default behavior is a recycle, not a delete.

Reply:
Yes, but if we call it RecycleNode then the decrement node pool size argument makes no sense any more. What if I add to the doc that by default, it will cause a new node to be created?

Reply:
FWIW, we identified _replace_ awhile ago as a decent compromise with regards to naming.